iron-autogrow-textarea.html

```diff
@@ -108,6 +105,7 @@
     <!-- size the input/textarea with a div, because the textarea has intrinsic size in ff -->
     <div class="textarea-container fit">
       <textarea id="textarea"
+        name$="[[name]]"
```

Comment:
Right now `value` gets set because `_onValueChanged -> _onBindValueChanged -> this.textarea.value`. 
Since `bindValue` is deprecated, it's good to have `value` bound here `value$=[[value]]`